pygsti/tools/symplectic.py

```diff
@@ -1026,7 +1038,7 @@ def compute_internal_gate_symplectic_representations(gllist=None):
     return srep_dict
 
 
-def symplectic_rep_of_clifford_circuit(circuit, srep_dict=None, pspec=None):
+def symplectic_rep_of_clifford_circuit(circuit, srep_dict={}, pspec=None):
```

Comment:
Let's switch this back to None, unless the overhead is really a problem.  Using an empty dictionary as a default argument is bad practice since a dict is a mutable type -- weird stuff can happen if things go awry.  This may mean the docstring gets reverted too?

Reply:
Not a problem at all, that's a good point. Changed it back.